app/assets/javascripts/oxalis/controller/annotations/skeletontracing_controller.js

```diff
@@ -1,80 +0,0 @@
-/**
- * skeletontracing_controller.js
- * @flow
- */
-
-import _ from "lodash";
-import Backbone from "backbone";
-import Store from "oxalis/store";
-import constants from "oxalis/constants";
-import { updateUserSettingAction } from "oxalis/model/actions/settings_actions";
-import { setActiveNodeRadiusAction } from "oxalis/model/actions/skeletontracing_actions";
-import { setPositionAction } from "oxalis/model/actions/flycam_actions";
-import { getActiveNode } from "oxalis/model/accessors/skeletontracing_accessor";
-import View from "oxalis/view";
-import SceneController from "oxalis/controller/scene_controller";
-import modal from "oxalis/view/modal";
-
-
-class SkeletonTracingController {
-
-  // See comment in Controller class on general controller architecture.
-  //
-  // Skeleton Tracing Controller:
-  // Add Skeleton Tracing controls that are not specific to the view mode.
-  // Also, this would be the place to define general Skeleton Tracing
-  // functions that can be called by the specific view mode controller.
-
-  skeletonTracingView: View;
-  sceneController: SceneController;
-
-  constructor(
-    skeletonTracingView: View,
-    sceneController: SceneController,
-  ) {
-    _.extend(this, Backbone.Events);
-    this.skeletonTracingView = skeletonTracingView;
-    this.sceneController = sceneController;
-  }
-
-
-  setParticleSize = (delta: number): void => {
-    let particleSize = Store.getState().userConfiguration.particleSize + delta;
-    particleSize = Math.min(constants.MAX_PARTICLE_SIZE, particleSize);
-    particleSize = Math.max(constants.MIN_PARTICLE_SIZE, particleSize);
-
-    Store.dispatch(updateUserSettingAction("particleSize", particleSize));
-  }
-
-
-  setRadius(delta: number): void {
-    getActiveNode(Store.getState().tracing)
-      .map(activeNode =>
-        Store.dispatch(setActiveNodeRadiusAction(activeNode.radius * Math.pow(1.05, delta))));
-  }
-
-
-  toggleSkeletonVisibility = (): void => {
```

Comment:
As far as I can see, this code was never used, which is why I didn't bother to reincorporate this somewhere. We might want to keep this in mind.